app/src/main/java/net/grandcentrix/blutufintegration/ui/list/ListViewModel.kt

```diff
@@ -10,20 +11,26 @@ import net.grandcentrix.blutuf.core.api.BlutufEventResult
 import net.grandcentrix.blutufintegration.data.model.DeviceUiState
 import net.grandcentrix.blutufintegration.data.repo.BluetoothRepository
 
+private const val SCAN_TIMEOUT: Long = 5000
+
 class ListViewModel : ViewModel() {
 
     val uiModel = BluetoothRepository.devicesStateFlow.asLiveData()
 
-    val selectedDevice: LiveData<DeviceUiState?> = BluetoothRepository.selectedDeviceStateFlow.asLiveData()
+    val selectedDevice: LiveData<DeviceUiState?> =
+        BluetoothRepository.selectedDeviceStateFlow.asLiveData()
 
     var bleStateFlow = MutableStateFlow(false)
 
+    val scanState = MutableLiveData(false)
```

Comment:
In most cases a `LiveData` is only observed by the view (e.g. Fragment). In those cases we should not expose that it is a `MutableLiveData` to outside of the ViewModel.
This can be done by this common pattern.
```suggestion
    private val _scanState = MutableLiveData(false)
    val scanState : LiveData<Boolean> = _scanState
```